assets/wizards/popups/utils.js

```diff
@@ -197,7 +236,10 @@ const sharesSegments = ( segmentsA, segmentsB ) => {
 export const warningForPopup = ( prompts, prompt ) => {
```

Comment:
There is no conflict message when prompts have no segment assigned:

![image](https://user-images.githubusercontent.com/7383192/112126013-502e6200-8bc4-11eb-99ce-bac550c37760.png)

Reply:
Ah, you're right—fixed in e6b29ed5